app/controllers/guides_controller.rb

```diff
@@ -97,6 +97,7 @@ def publish
     ActiveRecord::Base.transaction do
       @guide.latest_edition.update_attributes!(state: 'published')
       GuidePublisher.new(guide: @guide).publish
+      SearchIndexer.new(@guide).index
```

Comment:
Do we need to think about the exceptions this may raise, as it will roll back the transaction?